Makefile

```diff
@@ -36,10 +36,11 @@ endif
 
 # Basic machine detection
 HOST_MACHINE = $(shell uname -m)
-ifeq ($(HOST_MACHINE),x86_64)
-HOST_IS_X86_64 = true
+MACHINE ?= $(HOST_MACHINE)
```

Comment:
where do the names "HOST_MACHINE" and "MACHINE" come from? Are they standard or did you come up with those names? (they don't seem standard to me that's why I'm asking).

Reply:
> It would be good if we could get away without needing extra per-arch compilation flags altogether.
>
> 
> Absl says "ABSL_RANDOM_RANDEN_COPTS blaze copts flags which are required by each environment to build an accelerated RandenHwAes library" - but do we really need the said `RandenHwAes` library? How do we use it?
> https://github.com/abseil/abseil-cpp/blob/b315753c0b8b4aa4e3e1479375eddb518393bab6/absl/copts/configure_copts.bzl#L43

I agree! See the issues with introducing the SSE4.1 requirement in https://github.com/grpc/grpc/issues/27095. I think the compiler uses runtime dispatch for `-maes` (https://github.com/abseil/abseil-cpp/issues/643#issuecomment-602643378), but I don't believe that's the case for `-msse4.1`.

Can we just drop `-msse4.1` the flag?